src/main/groovy/com/bugsnag/android/gradle/Abi.groovy

```diff
@@ -1,5 +1,9 @@
 package com.bugsnag.android.gradle
 
+import groovy.transform.CompileStatic
+
+@SuppressWarnings('DuplicateStringLiteral')
```

Comment:
There are duplicate strings in this enum constructor, but they represent different concepts and shouldn't be grouped together into one variable.